### PR TITLE
Feature/initial development

### PR DESCRIPTION
**Pull request by gareth-e**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+# Install Oracle client packages from Oracle Linux yum server:
+RUN  yum -y install oracle-release-el7 && yum-config-manager --enable ol7_oracle_instantclient && \
+     yum -y install oracle-instantclient${release}.${update}-basic oracle-instantclient${release}.${update}-devel oracle-instantclient${release}.${update}-sqlplus oracle-instantclient${release}.${update}-tools && \
+     rm -rf /var/cache/yum
+
+# Path update required for tools package:
+ENV PATH=$PATH:/usr/lib/oracle/${release}.${update}/client64/bin
+
+# Additional packages used in scripts:
+RUN  yum install -y hostname-3.13-3.el7.x86_64 && \
```
Comment (marcransome) [inline Dockerfile]: The package command here has the potential to fail in future if the RPM release version changes (i.e. if the RPM is patched). Instead, specify the version without the release number and architecture, for example `hostname-3.13`, to ensure the software version is maintained but the latest RPM edition is preferred (often older RPMs aren't maintained for the same software version).

File: README.md [hunk 2]
```diff
@@ -1,2 +1,7 @@
 # ci-oracle-client-runner
-A container that contains an oracle client for running against Concourse 
+A container that contains an oracle client for running against Concourse.\
+The files here are based on the OracleInstantClient docker images provided by Oracle, that can be found here: https://github.com/oracle/docker-images/tree/master/OracleInstantClient \
```
Comment (marcransome) [inline README.md]: The information available at this URL is likely to change as it contains a mutable reference (i.e. `master`). Links to GitHub should ideally be permalinks or they quickly become outdated. Press `y` when viewing the page on GitHub to update the URL to a permalink and replace the above URL.

File: README.md [hunk 3]
```diff
@@ -1,2 +1,7 @@
...
+This is based on the Oracle Linux 7-slim (dist-amd64) container image: https://github.com/oracle/container-images/tree/573754fe56e7e8d570c554e8152fc9606c89561d/7-slim
+
+The Dockerfile is based on Oracle template: https://github.com/oracle/docker-images/blob/master/OracleInstantClient/dockerfiles/19/Dockerfile
```
Comment (marcransome) [inline README.md]: As above, there's a mutable reference in the URL here which means the page contents will change over time.

File: README.md [hunk 4]
```diff
@@ -1,2 +1,7 @@
 # ci-oracle-client-runner
-A container that contains an oracle client for running against Concourse 
+A container that contains an oracle client for running against Concourse.\
```
Comment (marcransome) [inline README.md:2]: The container isn't bound to Concourse in any way and could for example be used under another CI system that supports containers, so the description here could be more generic, something like: `Docker configuration for Oracle Instant Client.`

**Review by marcransome — CHANGES_REQUESTED** [review]
It's worth considering whether the links presented in the `README` would be better represented using Markdown inline-style links with titles. For example:

> The files here are based on [Docker images provided by Oracle](https://github.com/oracle/docker-images/tree/4598a9fe9dd0d4c19790f0bee8493e415263a083/OracleInstantClient)

Reads better than:

> The files here are based on the OracleInstantClient docker images provided by Oracle, that can be found here:  https://github.com/oracle/docker-images/tree/4598a9fe9dd0d4c19790f0bee8493e415263a083/OracleInstantClient

File: Dockerfile (hunk 1 above)
Comment (gareth-e) [reply Dockerfile]: Done.

File: README.md (hunk 2 above)
Comment (gareth-e) [reply README.md]: Done.

File: README.md (hunk 3 above)
Comment (gareth-e) [reply README.md]: Also updated.

File: README.md (hunk 4 above)
Comment (gareth-e) [reply README.md:2]: Yep, fair shout. Updated.

**Review by marcransome — APPROVED** [review]

**Pull request merged**